src/store/annotations/util.ts

```diff
@@ -0,0 +1,31 @@
+import * as uuid from 'uuid';
+import { Annotation } from '../../@types';
+
+export function addClientIds(annotation: Annotation): Annotation {
+    if (annotation.target.type !== 'drawing') {
```

Comment:
This seems very drawing-specific, but the function signature, module name, and types (Annotation) are generic. Should we rename things and refine the types?